download/_posts/2015-05-05-2.12.0-M1.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Scala 2.12.0-M1
+start: 05 May 2015
+layout: downloadpage
+release_version: 2.12.0-M1
+release_date: "May 05, 2015"
+show_resources: "true"
+permalink: /download/2.12.0-M1.html
+requirements: "This Scala software distribution can be installed on any Unix-like or Windows system. It requires the Java runtime version 1.6 or later, which can be downloaded <a href='http://www.java.com/'>here</a>."
+resources: [
+  ["-main-unixsys", "scala-2.12.0-M1.tgz", "http://downloads.typesafe.com/scala/2.12.0-M1/scala-2.12.0-M1.tgz", "Max OS X, Unix, Cygwin", "23.85M"],
+  ["-main-windows", "scala-2.12.0-M1.msi", "http://downloads.typesafe.com/scala/2.12.0-M1/scala-2.12.0-M1.msi", "Windows (msi installer)", "102.77M"],
+  ["-non-main-sys", "scala-2.12.0-M1.zip", "http://downloads.typesafe.com/scala/2.12.0-M1/scala-2.12.0-M1.zip", "Windows", "23.90M"],
+  ["-non-main-sys", "scala-2.12.0-M1.deb", "http://downloads.typesafe.com/scala/2.12.0-M1/scala-2.12.0-M1.deb", "Debian", "70.35M"],
+  ["-non-main-sys", "scala-2.12.0-M1.rpm", "http://downloads.typesafe.com/scala/2.12.0-M1/scala-2.12.0-M1.rpm", "RPM package", "101.61M"],
+  ["-non-main-sys", "scala-docs-2.12.0-M1.txz", "http://downloads.typesafe.com/scala/2.12.0-M1/scala-docs-2.12.0-M1.txz", "API docs", "43.72M"],
+  ["-non-main-sys", "scala-docs-2.12.0-M1.zip", "http://downloads.typesafe.com/scala/2.12.0-M1/scala-docs-2.12.0-M1.zip", "API docs", "81.12M"],
+  ["-non-main-sys", "scala-sources-2.12.0-M1.tar.gz", "https://github.com/scala/scala/archive/v2.12.0-M1.tar.gz", "Sources", ""]
+]
+---
```

Comment:
the file was generated by https://github.com/scala/make-release-notes.. i'll see if I can easily change it.